src/sas/qtgui/Utilities/Preferences/PreferencesPanel.py

```diff
@@ -30,6 +30,19 @@
 logger = logging.getLogger(__name__)
 
 
+def restart_main():
+    # Restart the application...
+    executable = sys.executable
+    executable_filename = os.path.split(executable)[1]
+    if executable_filename.lower().startswith('python'):
+        # application is running within a python interpreter
+        python = executable
+        os.execv(python, [python, ] + sys.argv)
```

Comment:
This line looks weird, are we sure its right? Should one be in quotes, just checking

Reply:
Both pythons are already strings, so no quotes needed, but the extra one is required for the restart when running from source.